src/utils.js

```diff
@@ -109,3 +109,32 @@ export function pFinally(promise, fn) {
     reason => Promise.resolve(fn()).then(() => { throw reason; })
   );
 }
+
+/**
+ * Simple deep object comparison function. This only supports comparison of
+ * serializable JavaScript objects.
+ *
+ * @param  {Object} a The source object.
+ * @param  {Object} b The compared object.
+ * @return {Boolean}
+ */
+export function deepEqual(a, b) {
+  if (a === b) {
+    return true;
+  }
+  if (typeof(a) !== typeof(a)) {
+    return false;
+  }
```

Comment:
Fixed in https://github.com/Kinto/kinto.js/commit/10b553d609fc8fd0e203ab14669b27550f774163